cookbook/test/integration/data_bags/secrets/aws_credentials.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "aws_credentials",
+  "Baragon": {
+    "encrypted_data": "FhcKQQBP4bzmgP0xVxgmJAjSpHReFgr6AcvLmwQZCoBGPC4aRorHXJYN0PLu\nnUk8e6+YWV8RENmYxT0o5LlQZ96QS/zXfeOfvT0uos+vUfSEWESwLKLVoVkb\nPccz3paJrnJYWRQM7aVlXVbewl/SKa4TbJARGqKTYrVlkRyM0IM=\n",
+    "iv": "eMpNP5SFXjLiDwMYUaLBIg==\n",
+    "version": 1,
+    "cipher": "aes-256-cbc"
+  }
+}
```

Comment:
Probably redundant to point this out but this will be removed along with switching to a maven artifact solution.